lib/rack/fake_cas.rb

```diff
@@ -77,6 +81,19 @@ def render_login_page
     [ 200, { 'Content-Type' => 'text/html' }, [login_page] ]
   end
 
+  def redirect_to(url)
+    [ 302, { 'Content-Type' => 'text/plain', 'Location' => url }, ['Redirecting you...'] ]
+  end
+
+  def render_status(status)
+    [ status, { 'Content-Type' => 'text/plain' }, [] ]
+  end
+
+  def ajax_request?
```

Comment:
Actual `.xhr?` implementation checks `get_header('HTTP_X_REQUESTED_WITH')` value - so that's the same here.

The other variant (`get_header('X-Requested-With')`) was necessary for specs to work - for some reason headers are handled differently there and I thought I can add this here, because it's only for development.

But probably I can adjust the specs instead of the actual code, let me try it ;)

Reply:
@ttruong21 Done. Thank you, it's better now ;)